### ISSUE TITLE
Suggestion for Skill Assignment

### ISSUE THREAD

**Issue by MageOhki** [MegaMek/mekhq#1411]
### Environment
Doesn't matter

### Description
Two fold:  First and foremost, have skills make sense by _era_.  3015 personnel should not have a lot with Tech or Gunnery BA, or any protomech skill.  Since only industrial exos exist, and Protomechs don't at all.  For secondary skills, perhaps an option to *weigh* what they have, or disallow at all?

**Comment by inqistor** [comment]
I second that. Even if you set - for all levels of the skill, people with that skill still shows up in Personnel Market.

**Comment by Windchild292** [comment]
@MageOhki and @inqistor
Which type of personnel market are you currently using? That is critical information for this issue.

**Comment by inqistor** [comment]
I don't think it is Market Issue, because you can not turn off skills. I started New Campaign under 47.9. Tech Limits are set to Limit by Year, and Disallow Extinct (I don't think that setting matters, but there are no Protomechs in 3023!). All Experience in all fields for Protomechs, Battlesuits, and even Tech/BA is set to -1.
I am using FM: Mercenaries Revised Personnel Market, and all those skills are showing up with standard random distribution (well, reduced because they are set in Skill Randomization as -2, or -3). I have even set 0 between +2, and +4 for all Techs in EXPERIENCE TABLE, but new people are showing up with 2, or 3 skill levels with no problem, so obviously Personnel Market Generation doesn't use that table at all.

Just give us something to turn off certain skills.

**Comment by Windchild292** [comment]
This is entirely market based, namely the extinction date is not checked in certain market types. I have taken a look at the base code and know how to do it, just haven't gotten to it yet.

As for the experience table, that is solely for the cost to increase, not for disabling skills. That functionality is not implemented and would be a beast to handle.
